CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ add_definitions(-DLOG_USE_COLOR)
 set(CMAKE_VERBOSE_MAKEFILE ON)
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -pthread -fPIC -Werror")
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} --std=c++14 -fpermissive -Wall -Wextra -lpthread")
+set(CMAKE_CXX_COMPILER g++52)
```

Comment:
Can you move this option to the cmake command line? Is g++52 a requirement? If not, adjust it to AppVeyor environment as well.

Reply:
I will remove it.